builder/builder-next/controller.go

```diff
@@ -254,3 +259,15 @@ func parsePlatforms(platformsStr []string) ([]specs.Platform, error) {
 	}
 	return out, nil
 }
+
+func getEntitlements(conf config.BuilderConfig) ([]string, error) {
+	var ents []string
+	// Incase of no config settings, NetworkHost should be enabled & SecurityInsecure must be disabled.
+	if conf.Entitlements.NetworkHost == nil || *conf.Entitlements.NetworkHost == true {
```

Comment:
Looks like the linter doesn't like this;

```
04:57:43  builder/builder-next/controller.go:269:50:warning: should omit comparison to bool constant, can be simplified to *conf.Entitlements.SecurityInsecure (S1002) (gosimple)
```

```suggestion
	if conf.Entitlements.NetworkHost == nil || *conf.Entitlements.NetworkHost {
```